frame/staking/src/testing_utils.rs

```diff
@@ -105,11 +136,12 @@ pub fn create_stash_and_dead_controller<T: Config>(
 pub fn create_validators<T: Config>(
 	max: u32,
 	balance_factor: u32,
+	seed: u32
 ) -> Result<Vec<<T::Lookup as StaticLookup>::Source>, &'static str> {
 	let mut validators: Vec<<T::Lookup as StaticLookup>::Source> = Vec::with_capacity(max as usize);
 	for i in 0..max {
 		let (stash, controller) =
-			create_stash_controller::<T>(i, balance_factor, RewardDestination::Staked)?;
+			create_stash_controller::<T>(i + seed, balance_factor, RewardDestination::Staked)?;
```

Comment:
we need this so we can call this function multiple times in the same logic branch